docs/deployment/azure-scripts/README.md

```diff
@@ -0,0 +1,53 @@
+# AKS ARM Template
+This Azure Resource Manager (ARM) template deploys an Azure Kubernetes Service (AKS) cluster with the specified parameters. The template also includes a deployment for installing Helm, a package manager for Kubernetes.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fshowwhy%2Fazure%2Fdocs%2Fdeployment%2Fazure-scripts%2Fall.json)
+
+**Please note that:**
+
+**1.    You must have:
+Microsoft.Authorization/roleAssignments/write permissions, such as `User Access Administrator` or `Owner`.**
+
+**2.    The aks cluster will be only accessible by Azure Portal, it's not generating a ssh key for external access.**
+
+**3.    It has self signed certificated for now, so when you access it will say 'Your connection is not private', you'll have to click on the advanced to proceed to the deployed website**
```

Comment:
"The application will have a self-signed certificate, when you access it you will se an error stating "Your connection is not private""